tests/Doctrine/fixtures/expected_xml/src/Repository/UserRepository.php

```diff
@@ -11,6 +11,9 @@
  * @method UserXml|null findOneBy(array $criteria, array $orderBy = null)
  * @method UserXml[]    findAll()
  * @method UserXml[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ * @method UserXml|null findOneByName($name, array $orderBy = null)
+ * @method UserXml[]    findByName($name, array $orderBy = null, $limit = null, $offset = null)
+ * @method int          countByName($name)
```

Comment:
Would this add 3 new `@method` lines for every property in the entity?

This is a really nice-looking PR, but that's a lot of noise (and I don't love this magic feature of Doctrine).